include/tables/scene_table.h

```diff
@@ -9,114 +9,114 @@
+/* 0x15 */ DEFINE_SCENE(FIRE_bs_scene, none, SCENE_FIRE_TEMPLE_BOSS, SDC_FIRE_TEMPLE, 0, 0)
+/* 0x16 */ DEFINE_SCENE(MIZUsin_bs_scene, none, SCENE_WATER_TEMPLE_BOSS, SDC_WATER_TEMPLE_BOSS, 0, 0)
+/* 0x17 */ DEFINE_SCENE(jyasinboss_scene, none, SCENE_SPIRIT_TEMPLE_BOSS, SDC_DEFAULT, 0, 0)
+/* 0x18 */ DEFINE_SCENE(HAKAdan_bs_scene, none, SCENE_SHADOW_TEMPLE_BOSS, SDC_SHADOW_TEMPLE, 0, 0)
+/* 0x19 */ DEFINE_SCENE(ganon_boss_scene, none, SCENE_GANONDORF_BOSS, SDC_DEFAULT, 0, 0)
+/* 0x1A */ DEFINE_SCENE(ganon_final_scene, none, SCENE_GANONS_CASTLE_COLLAPSE, SDC_GANONS_CASTLE_COLLAPSE, 0, 0)
+/* 0x1B */ DEFINE_SCENE(entra_scene, none, SCENE_MARKET_ENTRANCE_DAY, SDC_DEFAULT, 0, 0)
+/* 0x1C */ DEFINE_SCENE(entra_n_scene, none, SCENE_MARKET_ENTRANCE_NIGHT, SDC_DEFAULT, 0, 0)
+/* 0x1D */ DEFINE_SCENE(enrui_scene, none, SCENE_MARKET_ENTRANCE_RUINS, SDC_DEFAULT, 0, 0)
+/* 0x1E */ DEFINE_SCENE(market_alley_scene, g_pn_18, SCENE_BACK_ALLEY_DAY, SDC_DEFAULT, 0, 0)
+/* 0x1F */ DEFINE_SCENE(market_alley_n_scene, g_pn_18, SCENE_BACK_ALLEY_NIGHT, SDC_DEFAULT, 0, 0)
 /* 0x20 */ DEFINE_SCENE(market_day_scene, g_pn_17, SCENE_MARKET_DAY, SDC_DEFAULT, 0, 0)
 /* 0x21 */ DEFINE_SCENE(market_night_scene, g_pn_17, SCENE_MARKET_NIGHT, SDC_DEFAULT, 0, 0)
 /* 0x22 */ DEFINE_SCENE(market_ruins_scene, g_pn_17, SCENE_MARKET_RUINS, SDC_DEFAULT, 0, 0)
-/* 0x23 */ DEFINE_SCENE(shrine_scene, none, SCENE_SHRINE, SDC_DEFAULT, 0, 0)
-/* 0x24 */ DEFINE_SCENE(shrine_n_scene, none, SCENE_SHRINE_N, SDC_DEFAULT, 0, 0)
-/* 0x25 */ DEFINE_SCENE(shrine_r_scene, none, SCENE_SHRINE_R, SDC_DEFAULT, 0, 0)
-/* 0x26 */ DEFINE_SCENE(kokiri_home_scene, none, SCENE_KOKIRI_HOME, SDC_DEFAULT, 0, 0)
-/* 0x27 */ DEFINE_SCENE(kokiri_home3_scene, none, SCENE_KOKIRI_HOME3, SDC_DEFAULT, 0, 0)
-/* 0x28 */ DEFINE_SCENE(kokiri_home4_scene, none, SCENE_KOKIRI_HOME4, SDC_DEFAULT, 0, 0)
-/* 0x29 */ DEFINE_SCENE(kokiri_home5_scene, none, SCENE_KOKIRI_HOME5, SDC_DEFAULT, 0, 0)
-/* 0x2A */ DEFINE_SCENE(kakariko_scene, none, SCENE_KAKARIKO, SDC_DEFAULT, 0, 0)
-/* 0x2B */ DEFINE_SCENE(kakariko3_scene, none, SCENE_KAKARIKO3, SDC_DEFAULT, 0, 0)
-/* 0x2C */ DEFINE_SCENE(shop1_scene, g_pn_23, SCENE_SHOP1, SDC_DEFAULT, 0, 0)
+/* 0x23 */ DEFINE_SCENE(shrine_scene, none, SCENE_TEMPLE_OF_TIME_EXTERIOR_DAY, SDC_DEFAULT, 0, 0)
+/* 0x24 */ DEFINE_SCENE(shrine_n_scene, none, SCENE_TEMPLE_OF_TIME_EXTERIOR_NIGHT, SDC_DEFAULT, 0, 0)
+/* 0x25 */ DEFINE_SCENE(shrine_r_scene, none, SCENE_TEMPLE_OF_TIME_EXTERIOR_RUINS, SDC_DEFAULT, 0, 0)
+/* 0x26 */ DEFINE_SCENE(kokiri_home_scene, none, SCENE_KNOW_IT_ALL_BROS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x27 */ DEFINE_SCENE(kokiri_home3_scene, none, SCENE_TWINS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x28 */ DEFINE_SCENE(kokiri_home4_scene, none, SCENE_MIDOS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x29 */ DEFINE_SCENE(kokiri_home5_scene, none, SCENE_SARIAS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x2A */ DEFINE_SCENE(kakariko_scene, none, SCENE_KAKARIKO_CENTER_GUEST_HOUSE, SDC_DEFAULT, 0, 0)
```

Comment:
```suggestion
/* 0x2A */ DEFINE_SCENE(kakariko_scene, none, SCENE_KAKARIKO_GUEST_HOUSE, SDC_DEFAULT, 0, 0)
```

I don't think the word CENTER is necessary here. It doesn't add any extra information as for the purpose of the scene that isn't already expressed by just calling it GUEST_HOUSE, and KAKARIKO_GUEST_HOUSE is a unique enough name to distinguish it from the other scenes without confusion, while also accurately describing the purpose of the scene. Furthermore, I am against naming scenes in relative terms unless there is no other unique distinguishing feature, which is definitely not the case here. In most use cases, what is important about the scene is what is inside, not where it is located, so it should be named as such.

Reply:
This house isn't well established in the lore as a "guest house", so it's clearer to also state its location, as several people have expressed

I think stating the location is fine. "What if I change this" arguments can lead to not naming anything because everything can be changed beyond being recognizable
For example: what if I turn this house into the carpenter boss's house then "guest house" is wrong too, what if I move the house to Hyrule Field then "kakariko" is wrong...

Reply:
> "What if I change this" arguments can lead to not naming anything because everything can be changed beyond being recognizable.

But it's important to consider "where" the changes are being made, and how it affects the thing being named. Ideally, scenes should be named in such a way that changing something else isn't likely to lead to a name change, hence why relative names should be avoided.

If you edit the game so that say, Inside the Deku Tree appears when entering Jabu Jabu's mouth, it still looks like the inside of the Deku Tree, and plays the same. Similarly, if you edit the game to where entering Lakeside Lab from Lake Hylia takes you to the house, the house is still tied to Kakariko because the NPCs inside still talk as if they're in Kakariko Village. It's still a Kakariko house until you change the NPCs inside of it.

This isn't the case with calling it the CENTER house. Nothing about the scene itself demands that it be identified as being in the center of Kakariko, it's not a proper name like Lakeside Laboratory, it's just a house that happens to be relatively close to the middle of the map. But you can easily break that designation of being near the center by modifying Kakariko in a number of ways, without changing the function of the scene itself.

Reply:
Some think `_center_` is required to refer to this scene non-ambiguously,
and besides you there is no strong opposition to it,
so I think it's best we move forward with `SCENE_KAKARIKO_CENTER_GUEST_HOUSE`.

I won't be debating this further, but if someone else feels strongly against `_center_` as well, you are welcome to express yourself